main.py

```diff
@@ -100,6 +100,9 @@ def update_services(client: DockerClient, apprise: Apprise):
 
             success_message = f'Update successful. Took {elapsed} seconds.'
             apprise.notify(title=f'Service: `{name}`', body=success_message, notify_type=NotifyType.SUCCESS)
+            if getenv('ONE_BY_ONE_UPDATE','') == 'yes':
```

Comment:
`PARALLEL_UPDATES`

Reply:
As you've noticed this would require to inverse the logic. As default behaviour is to do parallel updates then it seems strange to introduce a variable to set it. Maybe UNPARALLEL_UPDATES is a better name?

Reply:
I don't think that is an issue. Documentation could mention that "parallel updates are turned on by default, to allow for faster updates of services. In order to turn them off, set the environment variable to 'off' or 'false'".